tests/test_all.py

```diff
@@ -421,6 +425,62 @@ def test_no_compat_packages(container):
         ).is_installed
 
 
+@pytest.mark.parametrize(
+    "container",
+    ALL_CONTAINERS,
+    indirect=True,
+)
+def test_bci_eula_is_correctly_available(container: ContainerData) -> None:
+    """Ensure that the BCI eula exists iff it's not a LTSS container"""
+
+    bci_license = "/usr/share/licenses/product/BCI/license.txt"
+    if OS_VERSION in ALLOWED_BCI_REPO_OS_VERSIONS and OS_VERSION in (
+        *RELEASED_SLE_VERSIONS,
+        "15.6",
+    ):
```

Comment:
`"15.6"` is already in `RELEASED_SLE_VERSIONS`:
```suggestion
    if OS_VERSION in ALLOWED_BCI_REPO_OS_VERSIONS and OS_VERSION in RELEASED_SLE_VERSIONS:
```